src/hardware/gus.cpp

```diff
@@ -665,27 +673,37 @@ void Gus::ActivateVoices(uint8_t requested_voices)
 	}
 }
 
-AudioFrame Gus::RenderFrame()
+const std::vector<AudioFrame>& Gus::GetFrames(const int num_requested_frames)
```

Comment:
Perhaps keep this `RenderFrames` so it's consistent with all the other audio device implementations (well, MT-32 and FluidSynth had only `Render`, but the rest `RenderFrames`). The fact that this function forwards the rendering job to the `Voice` classes is an implementation details, it should not be reflected in the function's name.